### ISSUE TITLE
99% of Examples are using React

### ISSUE THREAD

**Issue by dantrevino** [blockstack/docs#935]
In the real world, other ways of building apps exist.  It would be nice to have more generic examples instead of the hyper-focused on react-specific examples.

**Comment by agraebe** [comment]
agreed! we just recently added some content for non-react usage of Connect. What other parts should be reviewed? Do you want to help with a PR as well?

**Comment by markmhx** [comment]
I'm going to close this since we've been moving away from React and https://github.com/blockstack/docs/pull/1014 should get us mainly away from it as a default. Please re-open once that PR is merged if there are more concerns about React reliance then.
